.env.example

```
@@ -55,6 +55,13 @@ APP_URL=https://your-app.replit.dev
 # Uses ECS task role credentials — no key/secret needed in ECS
 SES_REGION=us-east-1
 
+# ML Zone Detection — Docling sidecar
+DOCLING_SERVICE_URL=http://localhost:8100
+
+# ML Zone Detection — pdfxt cloud service
+PDFXT_SERVICE_URL=https://app.pdfxt.com
+PDFXT_API_KEY=your_pdfxt_api_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PDFXT_API_KEY key should go before the PDFXT_SERVICE_URL key

(UnorderedKey)